core/src/main/scala-3/chisel3/VerificationStatementMacros.scala

```diff
@@ -4,6 +4,9 @@ package chisel3
 
 import chisel3.experimental.SourceInfo
 import chisel3.internal._
+import chisel3.internal.firrtl.ir._
```

Comment:
Nit: is this emitting that much from this that it needs a wildcard? Can this use a more targeted import?